SWIG/interpolation.i

```diff
@@ -174,6 +174,14 @@ class DefaultLogCubic : public QuantLib::LogCubic {
     : QuantLib::LogCubic(QuantLib::CubicInterpolation::Kruger) {}
 };
 
+class LogCubic : public QuantLib::LogCubic {
```

Comment:
This is the same as `SplineCubic` a few lines above.  I think we didn't use the `LogCubic` name because some compilers got confused between that and `QuantLib::LogCubic`.

Reply:
Okay I see what you mean.  I'll remove this change and just use class SplineCubic.  Thanks.